turn/rootfs/defaults/docker-entrypoint.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/ash
+# create config dir if not exists
+[ ! -d /config ] && mkdir /config
```

Comment:
No need IIRC, docker does this because /config is a volume.

Reply:
ask